storage/shard.go

```diff
@@ -1714,9 +1721,24 @@ func (s *dbShard) Flush(
 			return false
 		}
 
+		// Debug
+		if flushOutcome == series.BlockDoesNotExist {
+			numBlockDoesNotExist++
+		}
+		if flushOutcome == series.StreamDoesNotExist {
+			numStreamDoesNotExist++
+		}
+
 		return true
 	})
 
+	// Debug
+	s.logger.WithFields(
+		xlog.NewField("shard", s.ID()),
+		xlog.NewField("numBlockDoesNotExist", numBlockDoesNotExist),
+		xlog.NewField("numStreamDoesNotExist", numStreamDoesNotExist),
+	).Debug("shard flush outcome")
```

Comment:
Do we need this still?

Reply:
Yeah I think its useful to have it available without writing new builds

Reply:
spoke in person, gonna keep these in